docs/governance/grants-bounties/udt-grantee-handbook.md

```diff
@@ -10,80 +10,89 @@ sidebar_position: 2
+## Encourage conversation around the grant application
 
-The Grants Administrator will tally the scores given by the reviewers, average them and assign you to the proper grant level (# of UDT governance tokens) based on where you fall. For more information on how your application will be reviewed and how the levels are set, see the [“Grants Review Guide”](https://www.notion.so/Grant-Review-Guide-b02facce97164bec8652da9976da6418).
+Once submitted for review, grant applications can be viewed and commented on by anyone in the Unlock community. Proposers should encourage and engage in discussion about the grant application in the comments on the application itself on Github.
 
-## Setting Milestones
+![grant-4](https://user-images.githubusercontent.com/214760/229210991-73215c4d-8728-4523-ae9d-243129e31d2a.png)
+
+***A NOTE ON VOTING: There are two primary kinds of voting mechanisms in the Unlock DAO. “Off-chain” voting, where Unlock DAO members can vote to communicate their positions on a proposal, and “on-chain” voting, where the votes not only communicate positions, but also result in automatic execution of the proposal if the proposal passes. These methods and their distinctions will be important in the following sections.***
+
+
+## Before starting work, take a temperature check from the Unlock DAO community
+
+Before spending significant time or effort building in anticipation of receiving a grant, the applicant should strongly consider an off-chain “temperature check” proposal and vote using a system like [Snapshot](https://snapshot.org/#/unlock-protocol.eth). To do this, the applicant would create a proposal and encourage community members to indicate how they would likely vote on the proposal were it an on-chain vote. While the Snapshot vote is technically non-binding, it can give an indication of the interest in the grant application and a directional likelihood if the proposal would pass if proposed on-chain, and if the project proposed in the grant were completed as proposed.
+
+Taking an off-chain temperature check vote is optional, but it is recommended. Strong support from the community is an indicator that if the project were to be executed, the community would likely support the funding of the grant upon completion of the work.
+
+You can think of this temperature check as a “social contract.” If the DAO later rejects on-chain proposals that had previously been “approved” off-chain via Snapshots, it would likely chill the interest of future applicants in even submitting grant applications, since prospective applicants may not trust the fact that the ghrant payouts would be officially ratified once the on-chain vote is proposed in the future. 
```

Comment:
```suggestion
You can think of this temperature check as a “social contract.” If the DAO later rejects on-chain proposals that had previously been “approved” off-chain via Snapshots, it would likely chill the interest of future applicants in even submitting grant applications, since prospective applicants may not trust the fact that the grant payouts would be officially ratified once the on-chain vote is proposed in the future. 
```